src/ViewInterface.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Starlit App.
+ *
+ * @copyright Copyright (c) 2016 Starweb AB
+ * @license   BSD 3-Clause
+ */
+
+namespace Starlit\App;
+
+use Symfony\Component\HttpFoundation\Request;
+use Starlit\App\ViewHelper\AbstractViewHelper;
+
+/**
+ * Abstract implementation of a View
+ */
+interface ViewInterface
+{
+    /**
+     * Constructor.
```

Comment:
Many unnecessary comments in this file. Would be nice to keep it a bit more slim and only comment where something is unclear or requires some additional info